src/verso-manual/VersoManual/InlineLean.lean

```diff
@@ -168,6 +168,16 @@ def reportMessages {m} [Monad m] [MonadLog m] [MonadError m]
     if messages.hasErrors then
       throwErrorAt blame "No error expected in code block, one occurred"
 
+/--
+Produces, within the DocElabM, a Syntax expression that denotes the `hsl` value. Specifically,
```

Comment:
```suggestion
Produces the syntax of an expression that denotes the `hls` value. Specifically,
```
Docstrings are always paired with signatures, whether in hovers or online docs, so no need to mention the monad here.